src/components/AppProvider/config.ts

```diff
@@ -1 +0,0 @@
-export const needsVariantList = ['topBar'];
```

Comment:
@AndrewMusgrave are we no longer using this config to determine which components need a theme?

Reply:
This file has moved to `ThemeProvider`, I probably forgot to delete it. Either way safe to 🔥